### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by rolandlo**
I have added most changes since version 1.0.18 got released. The changelog for version 1.0.18 is still missing.

### CODE REVIEW

**Comment by Febbe** [comment]
The change log for the 1.0.18 release is on the 1.0.x branch

**Comment by rolandlo** [comment]
Should I just copy it from there to include it in the master branch?

**Comment by Technius** [comment]
The master changelog should be a diff against latest stable, so you'll want to wait until we merge #2071 and then we can proceed here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,23 +3,113 @@
...
+      or play (#1573)
 * Input System
-    * The new input system is now enabled by default. Should errors occur please
+    * The new input system is now enabled by default (#1560). Should errors occur please
       file a bug report and deactivate it in the settings.
-    * Zoom gesture reimplemented for better compatibility
+    * Zoom gesture reimplemented for better compatibility (#1528)
 * LaTeX
-    * Added support for `\newline`
+    * Added support for `\newline (#1469, thanks to @nicolae-stroncea)
```
Comment (nicegamer7) [inline CHANGELOG.md:19]: Forgot to close this code block.

**Comment by Technius** [comment]
@rolandlo If you're okay with this, I can apply the suggested fix and then merge this over the weekend, since I have some time.

**Comment by rolandlo** [comment]
@Technius Sure, I am fine with that.

**Comment by Technius** [comment]
I noticed that we're missing a lot of changes, so I just updated it for now with some of the ones I remember. I also reworded and reorganized the lines. At some point, we'll have to go back and look through the commits from `1.0.15`, since that is when we forked out `1.0.x` into a separate branch.

**Comment by rolandlo** [comment]
We will also have to update the file `debian/changelog`. This one is used for automatic versioning the [launchpad PPA releases](https://launchpad.net/~andreasbutti/+archive/ubuntu/xournalpp-master). The numbers 1.6.0-0 in those releases come from the fact that we haven't updated that file since version 1.0.6.

**Comment by LittleHuba** [comment]
We should include the changes for v1.0.20 as well and merge it before release.

**Comment by LittleHuba** [comment]
> We should include the changes for v1.0.20 as well and merge it before release.

Well that didn't work... @rolandlo could you add the last few backports you did and then merge this?

**Comment by rolandlo** [comment]
I made a mistake and have to work later on it.

**Pull request merged**